index.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * A quick canvas test to see if my math was correct for creating an ellipse from two points
+ * @author Dawn Moore
+ */
+
+/**
+ * A type containing the X and Y position of a given point on a graph
+ * positionX The x position of this point
+ * positionY the y position of this point
+ */
+interface Point {
+    positionX: number;
+    positionY: number;

```

Comment:
Simpler names here is fine, x and y, having to read 4x positionX and positionYs on one line is very verbose

Reply:
I'll make that edit now, I figured it was bad to have such a short variable. Thank you

Reply:
Short variable names ARE bad, but only if they're in a context where they can't be figured out at a glance. If a function is taking a point object and one sees `point.x` and `point.y`, everyone knows what these are without a more verbose name.